server/services/placeOnReportService.ts

```diff
@@ -156,17 +159,25 @@ export default class PlaceOnReportService {
         value: getFormattedOfficerName(reporter.name),
       },
       {
-        label: 'Date',
+        label: 'Date of Incident',
```

Comment:
incident should be lower looking at the wireframe.  Design wise i think Anne always prefers upper case on first word of sentence only

Reply:
Yep Tim is correct

Reply:
addressed throughout